python/vescale/dtensor/sharding_spec.py

```diff
@@ -0,0 +1,149 @@
+################################################################################
```

Comment:
I thinks this class have been removed. https://github.com/volcengine/veScale/pull/23